### PR TITLE
Locate main sheet. Now have to add a property to the JSON though.

### PR DESCRIPTION
**Pull request by malee31**
I don't know how to test it locally so just pray. Works in regular node at least.

### CODE REVIEW

File: src/state/DB.js [hunk 1]
```diff
@@ -167,6 +167,14 @@ class DB {
         );
     }
 
+	 getMainSheet(info) {
+			for (var sheetIndex = 0; sheetIndex < info.worksheets.length; sheetIndex){
+				if(!this.config.sheets.worksheet_name) break;
```
Comment (kaijchang) [inline src/state/DB.js:172]: can you add the `worksheet_name` property in https://github.com/Team4159/CardinalHours/blob/3644172bbe7ed649be1ef8d1fd532184490bbf7f/src/state/default_config.json#L1-L12
and remove the `worksheet_id` property there.

**Review by kaijchang — CHANGES_REQUESTED** [review]

File: src/state/DB.js [hunk 2]
```diff
@@ -167,6 +167,14 @@ class DB {
         );
     }
 
+	 getMainSheet(info) {
+			for (var sheetIndex = 0; sheetIndex < info.worksheets.length; sheetIndex){
```
Comment (kaijchang) [inline src/state/DB.js]: do you ever increment `sheetIndex`?

File: src/state/DB.js [hunk 3]
```diff
@@ -167,6 +167,14 @@ class DB {
         );
     }
 
+	 getMainSheet(info) {
```
Comment (kaijchang) [inline src/state/DB.js:170]: can you get the indentation to match

**Review by kaijchang — CHANGES_REQUESTED** [review]

File: src/state/DB.js (hunk 1 above)
Comment (malee31) [reply src/state/DB.js:172]: Should I keep id as a backup?

Comment (malee31) [reply src/state/DB.js:172]: Scratch that, It dies.

Comment (malee31) [reply src/state/DB.js:172]: Should I keep it in the code for outdated?

File: src/state/DB.js (hunk 3 above)
Comment (malee31) [reply src/state/DB.js:170]: It lines up for me

Comment (kaijchang) [reply src/state/DB.js:170]: i think you're using tabs while it uses spaces or vice versa

**Review by kaijchang — APPROVED** [review]
sure

**Pull request merged**